### PR TITLE
Please review this, @casey-collab.

### PR DESCRIPTION
**Pull request by rajendraranabhat**
Added my region folder and added message in my native language.

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: Nepalese/namaste.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+बंधु नमस्कार।
```
Comment (casey-collab) [inline Nepalese/namaste.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by rajendraranabhat** [comment]
I just added link for the nepali language. English translation of the word means "Hello how are you!!"

**Comment by rajendraranabhat** [comment]
You may merge now !!

**Comment by cbuckey-uda** [comment]
Sorry, @rajendraranabhat, I think you misunderstood. Your page doesn't need to contain a link, but there should be a link in `language.md` saying something like "If you speak English, go here", only instead of English, it should say your language, and it should be written in your language.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**